content/use/tab-commercial-cloud.md

```diff
@@ -2,5 +2,5 @@
 title: Commercial Cloud Services
 ---
 
-These are commercial cloud providers where ready to run Galaxy instances are available. You need to pay for these, but you only need to pay for as long as you need the instance to be up.
+These are commercial cloud providers ready to run where Galaxy instances are available. You need to pay for these, but you only need to pay for as long as you need the instance to be up.
```

Comment:
```suggestion
These are commercial cloud providers where ready-to-run Galaxy instances are available. You need to pay for these, but you only need to pay for as long as you need the instance to be up.
```

Reply:
Sorry but this sentence is not making sense to me. I changed the place of "where" because on the other document I changed (Academic Cloud Services) it talks about geographical location.

Reply:
The way I understand it (but I'm not an expert here) is: you pay a commercial provider to get access to the cloud, and you can pick an already-prepared Galaxy instance and run it easily. But I'm not super sure 🙂  @afgane maybe you can help with this sentence?

Reply:
ohhkayyy 😀 Yeah I think you are right. It made sense when I read it that way. Thank you for clarifying.

Reply:
ok, I'll merge then :) Thanks a lot @Melkeb!